examples/flutter_gallery/lib/demo/shrine/shrine_page.dart

```diff
@@ -88,6 +88,8 @@ class ShrinePageState extends State<ShrinePage> {
       appBar: new AppBar(
         elevation: _appBarElevation,
         backgroundColor: Theme.of(context).cardColor,
+        iconTheme: IconTheme.of(context),
```

Comment:
As a developer you basically never want to use IconTheme.of(). Only Icon and ImageIcon should use those (and anyone else pretending to be an Icon).

I agree we should doc that better.